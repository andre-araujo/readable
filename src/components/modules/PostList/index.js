import React from 'react';

import PostCard from '../../elements/PostCard';

function PostList() {
  return (
    <ul>
      <li>
        <PostCard
          id="id"
          timestamp="timestamp"
          title="title"
          body="body"
          author="author"
          category="category"
          voteScore="voteScore"
        />
      </li>
    </ul>
  )
}

export default PostList;
