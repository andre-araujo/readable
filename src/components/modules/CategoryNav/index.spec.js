import React from 'react';
import { shallow } from 'enzyme';

import PostList from '.';

describe('PostList', () => {
  it('should match snapshot', () => {
    expect(shallow(<PostList />)).toMatchSnapshot();
  });
});
