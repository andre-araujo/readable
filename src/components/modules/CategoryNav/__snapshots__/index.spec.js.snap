// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostList should match snapshot 1`] = `
<ul>
  <li>
    <PostCard
      author="author"
      body="body"
      category="category"
      id="id"
      timestamp="timestamp"
      title="title"
      voteScore="voteScore"
    />
  </li>
</ul>
`;
